functional/keylime_create_policy-static-data/main.fmf

```diff
@@ -0,0 +1,18 @@
+summary: Tests the keylime_create_policy tool
+description: |
+    Provide different types of input to the keylime_create_policy tool and
+    ensure that it produces the expected runtime policy.
+contact: Stefa Berger <stefanb@linux.ibm.com>
+component:
+  - keylime
+test: ./test.sh
+framework: beakerlib
+tag:
+  - CI-Tier-1
+require:
+  - jq
+recommend:
+  - keylime
+duration: 1m
```

Comment:
In case this test won't finish within ~15 seconds it may be safer to set the duration to 2 minutes.

Reply:
Fixed.